usr/share/lib/img_proof/tests/SLES/test_sles_smt_reg.py

```diff
@@ -15,7 +15,8 @@ def test_sles_smt_reg(
 
     # ensure the correct smt/rmt ip is used
     servers = get_smt_servers(provider)
-    smt_ips = [server['ip'] for server in servers]
+    smt_ips = list(filter(None, [server['ip'] for server in servers]))
+    smt_ips += list(filter(None, [server['ipv6'] for server in servers]))
```

Comment:
Won't that append a list as the last element of the first list?

e.g. `['1.2.3.4', '1.2.3.5', '1.2.3.6', ['11:22:33:44:55:66'...]]`

Reply:
No, I believe it's treated as a list merge

```
>>> t = ['a', 'b', 'c']
>>> s = ['d', 'e', 'f']
>>> t += s
>>> t
['a', 'b', 'c', 'd', 'e', 'f']
```

Reply:
And the filter removes anything that's "Nonesy" such as empty strings.